src/js/share/share.js

```diff
@@ -105,15 +105,15 @@ export class Share {
     this.workspace.clear();
     let svg = undefined;
     try {
-      jsonDomToWorkspace(blockJSON, this.workspace);
+      const sceneWidth = jsonDomToWorkspace(blockJSON, this.workspace);
       zebraChangeColor(this.workspace.topBlocks_);
       const oriSvg = this.workspace.getParentSvg();
       const oriBox = oriSvg.lastElementChild.getBBox();
 
       // remove rect around it
       svg = oriSvg.cloneNode(true);
       svg.lastElementChild.removeChild(svg.lastElementChild.firstElementChild);
-      svg.setAttribute('width', `${oriBox.width}px`);
+      svg.setAttribute('width', `${sceneWidth * this.config.renderSize}px`);
```

Comment:
Set width of catblocks-svg element to the with of the biggest brick. So scrolling the x axis works on Chrome and Firefox.